README.md

```diff
@@ -11,9 +11,9 @@ This action sets up a go environment for use in actions by:
 - optionally downloading and caching a version of Go by version and adding to PATH
 - registering problem matchers for error output
 
-# V2
+# V3
 
-The V2 offers:
+The V3 offers:
 - Adds GOBIN to the PATH
 - Proxy Support
 - `stable` input 
```

Comment:
This has actually been removed with V3: https://github.com/actions/setup-go/releases/tag/v3.0.0

Reply:
Thank you Brian for pointing this out :)

Reply:
Fixed, thanks!